### DISCUSSION TITLE
Visning af dropdown menu ifm. notifikation

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#32] [Bugs] [answered]
Når Malene og Jens sender reminder ud til de frivillige – bruger de funktionaliteten Nyhedsbreve.

Her står den default til at vælge alle datoer i event…og derfor skal Jens/Malene fjerne alle flueben manuelt undtagen den relevante dato hvor de skal sende reminder. 

Men når de skal fjerne alle flueben så ”falder” dropdown menu ned under skærmbilledet…og ja det gør det besværligt at fjerne alle disse flueben.

Kan I lave følgende fix?:
- Visning af dropdown menu skal fixes.  Den må ikke "falde udenfor skærmen". (bug)
- Det skal være default, at den sætter flueben i alle datoer. Lige nu er default ingen flueben. (enhancement)

**Comment by mikkelsl** [comment, marked as answer]
Har lavet en opgave her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=66391904
